### ISSUE TITLE
exceptions import

### ISSUE THREAD

**Issue by larsclaussen** [encode/requests-async#17]
First of all, good work there on the ASGI/async front! Highly appreciated.   

So far porting my code from ``requests`` to ``requests_async`` has been seamless. I was wondering, however, how to properly import exceptions. Using requests directly I'd do
```   
from requests import exceptions
```
I would expect that I was also able to import exceptions the same way
```   
from requests_async import exceptions
```
Is this by design because the intention is just to be able to use await for making requests? Or is the plan to wrap the requests lib entirely?

**Comment by lovelydinosaur** [comment]
Released as 0.1.2 👍

**Comment by larsclaussen** [comment]
Nice, thanks!
